diff --git a/config.py b/config.py
index b0e194e..d0ed90a 100644
--- a/config.py
+++ b/config.py
@@ -250,7 +250,7 @@ class Config(object):
         config = configparser.ConfigParser()
         if os.path.exists(filename):
             with codecs.open(filename, encoding='utf-8') as fp:
-                config.readfp(fp)
+                config.read_file(fp)
 
         return cls._parse_ttrv_file(config)
 
diff --git a/theme.py b/theme.py
index af662df..9cc7fae 100644
--- a/theme.py
+++ b/theme.py
@@ -399,7 +399,7 @@ class Theme(object):
             config = configparser.ConfigParser()
             config.optionxform = six.text_type  # Preserve case
             with codecs.open(filename, encoding='utf-8') as fp:
-                config.readfp(fp)
+                config.read_file(fp)
         except configparser.ParsingError as e:
             raise ConfigError(e.message)
 
